src/selectivity-base.js

```diff
@@ -207,6 +211,12 @@ $.extend(Selectivity.prototype, EventDelegator.prototype, {
         return this.$el.find(selector);
     },
 
+    _blurClose: function() {
+
+        if(!this.$el.hasClass('hover') && !this._opening) {
```

Comment:
Not completely sure if we need !this._opening check here. My assumption was that there can happen delayed focus events in the code while the select is opening while user moves the cursor out of the select area. 

Is there even possibility that there can be delayed focus events when the select is opening?